bundles/org.smarthomej.binding.viessmann/src/main/resources/update/device.update

```diff
@@ -28,3 +28,18 @@
 2;REMOVE_CHANNEL;heatingDhwTemperatureHysteresis
 3;REMOVE_CHANNEL;heatingDhwTemperatureTemp2
 3;REMOVE_CHANNEL;heatingDhwTemperatureMain
+3;REMOVE_CHANNEL;heatingCircuits0OperatingProgramsComfort#temperature
```

Comment:
Are these channels no longer needed?

Reply:
No, these channels only need to be recreated because the channel properties have changed